src/pythonjsonlogger/core.py

```diff
@@ -161,7 +163,9 @@ def __init__(
             style: how to extract log fields from `fmt`
             validate: validate `fmt` against style, if implementing a custom `style` you
                 must set this to `False`.
-            defaults: ignored - kept for compatibility with python 3.10+
+            defaults: a dictionary containing default values for unspecified
+                extras. {"key": 1234} will add the key to the json if
+                unspecified in the extras while logging a message.
```

Comment:
I'm wondering if we need to add a note that "these fields are added prior to renaming".